llvm/test/CodeGen/SPIRV/hlsl-intrinsics/WaveReadLaneAt.ll

```diff
@@ -0,0 +1,56 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv32v1.3-vulkan-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv32v1.3-vulkan-unknown %s -o - -filetype=obj | spirv-val %}
```

Comment:
Sorry I missed this in my review. The is the wrong triple to use. You should be using `spirv` not `spirv32`.

Reply:
I don't think anything else needs to be updated. I'll open a PR to fix this myself.

Reply:
https://github.com/llvm/llvm-project/pull/122738